flake.nix

```
@@ -240,5 +248,23 @@
             };
           };
       }
-    );
+    ) //
+    {
+      nixosModule = { config, ... }:
+        {
+          config = {
+            boot.kernelPatches = [{
+            name = "memflow-enable-kallsyms";
+            patch = null;
+            extraConfig = ''
+                KALLSYMS y
+                KALLSYMS_ALL y
+              '';
+            }];
+
+            boot.kernelModules = [ "memflow" ];
+            boot.extraModulePackages = [ (self.memflow-kmod.${builtins.currentSystem} config.boot.kernelPackages.kernel) ];
```

`builtins.currentSystem` is impure
```suggestion
            boot.extraModulePackages = [ (self.memflow-kmod.${pkgs.system} config.boot.kernelPackages.kernel) ];
```